.github/workflows/release-pipeline.yaml

```diff
@@ -63,6 +63,9 @@ jobs:
         runs-on:
           - ubuntu-latest
           - windows-latest
+        hg-version:
```

Comment:
this matrix value isn't being used anywhere. I think you need to pass it in on the `with` section a few lines down. Look at where `matrix.runs-on` is being used.

Reply:
Done in commit b8e77f1e.